src/angular-app/languageforge/lexicon/settings/configuration/configuration-fields.component.ts

```diff
@@ -72,6 +72,7 @@ export class FieldsConfigurationController implements angular.IController {
   overrideAll = ConfigurationFieldUnifiedViewModel.overrideAll;
   resetInputSystemsForRole = ConfigurationFieldUnifiedViewModel.resetInputSystemsForRole;
   resetInputSystemsForGroup = ConfigurationFieldUnifiedViewModel.resetInputSystemsForGroup;
+  requiredConfigurationFields = ConfigurationFieldUnifiedViewModel.disableRequiredFields;
```

Comment:
name change: remove "disabled"
```suggestion
  ensureRequiredConfigurationFields = ConfigurationFieldUnifiedViewModel.ensureRequiredFields;
```